geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSource.scala

```diff
@@ -100,6 +100,12 @@ class LiveKafkaConsumerFeatureSource(entry: ContentEntry,
     }
   })
 
+  val ses = new ScheduledThreadPoolExecutor(1)
+
+  ses.scheduleAtFixedRate(new Runnable() {
```

Comment:
do we need to run this if the cache is not set to auto-expire entries based on timeout? That is configurable via the expirationPeriod option.